user_research/Device/digitalData.md

```diff
@@ -0,0 +1,6 @@
+# 2012 AEC![devices](../Pictures/Devices.png)
+![ageInternet](../Pictures/ageAppropriation.png)
+![ageSmartphone](../Pictures/smartphoneAge.png)
+![mobinautes](../Pictures/mobinautes.png)
+# 2011 CREDOC
+![deviceEvolution](../Pictures/repartitionDeviceFoyerCREDOC.png)
```

Comment:
Why this graph contains data up to 2013 when it's supposed to be from 2011 ?

Plus, i understand that according to this 2011 (or 2013 ??) CREDOC poll, smartphone equipment rate is about 39% (for 12+ year olds), and according to AEC 2012 poll, smartphone equipment rate is about 26%. Who should we trust ? Maybe the difference comes from ages considered: AEC could be taking into consideration all population, babies included, which doesn't make sense. If so, CREDOC numbers could be more interesting.